firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/FirebaseAppDistributionNotificationsManager.java

```diff
@@ -81,60 +91,21 @@ private NotificationManager createNotificationManager(Context context) {
       channel.setDescription(context.getString(R.string.notifications_channel_description));
       // Register the channel with the system; you can't change the importance
       // or other notification behaviors after this
-      NotificationManager notificationManager = context.getSystemService(NotificationManager.class);
+      NotificationManager notificationManager =
+          (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
```

Comment:
This seems to be the more common way of getting a `NotificationManager`.

Reply:
Awesome! Do you mind linking to where else you have seen this?

Reply:
First of all I just noticed we're already using the String constant method on line 99 below.

Using the String constant is used pretty overwhelmingly throughout the repo: https://github.com/firebase/firebase-android-sdk/search?q=getSystemService

But for the NotificationManager specifically, using the class actually seems more common. So I don't know, it doesn't really matter I guess. I'm inclined to change it to at least make it consistent with line 99.

Reply:
Ah good catch making it more consistent